ethereum/forge-scripts/DeployTestToken.s.sol

```diff
@@ -95,11 +98,31 @@ contract DeployTestToken is Script {
         // mint 1000 units
         accountantToken.mint(accounts[9], 1_000_000_000_000_000_000_000);
 
+        // Deploy a test token for Transfer Verification
+        ERC20PresetMinterPauser deployedA = new ERC20PresetMinterPauser(
+            "TransferVerifier Test Token A",
+            "TVA"
+        );
+
+        console.log("Test token A deployed at: ", address(deployedA));
+
+        // Mint test tokens to Anvil's default account at index 1.
+        // The account at index 0 is used for other tests in the devnet, so
```

Comment:
account 1 is also used by the nft integration test [here](https://github.com/wormhole-foundation/wormhole/blob/883d94a92c2e6654d7414237f5c8bae7da57e038/sdk/js/src/nft_bridge/__tests__/utils/consts.ts#L11), as are many of the keys, which is why the anvil [accounts flag](https://github.com/wormhole-foundation/wormhole/blob/883d94a92c2e6654d7414237f5c8bae7da57e038/devnet/eth-devnet.yaml#L42) is set to 13. try searching the private keys (listed by anvil on startup) to ensure they are not used or bump the accounts flag to 14 and use the new key.

Reply:
Got it, I've updated these tests to use account[13] and added some comments